subsys/partition_manager/pm.yml.trustzone

```diff
@@ -4,5 +4,9 @@ sram_secure:
   region: sram_primary
 
 sram_nonsecure:
-  span: [sram_primary] /* Here, sram_primary refers to the (NS) app's RAM. */
   region: sram_primary
+  span:
+    - sram_primary /* Here, sram_primary refers to the (NS) app's RAM. */
+#ifdef CONFIG_BT_RPMSG_NRF53
+    - rpmsg_nrf53_sram
+#endif
```

Comment:
Hi @sigvartmh , @hakonfam ,

two questions.

is autoconf.h always included ?

What happens if a partition that we span over does not exist? Will that be silently ignored?

If so it is safer to unconditionally add rpmsg_nrf_sram, right?

Relates to

https://github.com/nrfconnect/sdk-nrf/pull/6958#issuecomment-1047601901

Reply:
Hi, I think the assumption that CONFIG_BT_RPMSG_NRF53 is always enabled when we need rpmsg_nrf53_sram is wrong.

At least this build:

~/ncs/nrf/samples/openthread/cli$ west build -p -b nrf5340dk_nrf5340_cpuapp_ns

does not enable

CONFIG_BT_RPMSG_NRF53

but it does have multiple images and needs a non-secure rpmsg area to communicate between them.

Reply:
1. Autoconf must be included explicitly by the pm.yml files.
2. It will be silently ignored

If you unconditionally span `rpmsg_nrf_sram` then whenever that partition exists it will be attempted to be spanned. The attempt will fail if it makes the span non-contiguous. You could argue that this is "safer" if the case will always be that it should be spanned by `sram_nonsecure`.

Reply:
Thank you